community.md

```diff
@@ -24,5 +24,8 @@ Below is a list of projects and resources from the Blazor community. If you know
 * [Blazor local and session storage support](https://github.com/cloudcrate/BlazorStorage)
 * [BlazorComponents (ChartJS Interop)](https://github.com/muqeet-khan/BlazorComponents)
 
+## Bootstrap components
```

Comment:
How about we rename the previous section to just "Libraries" and move BlazorComponents into that section?

Reply:
Yeah , that will be nice to have all interop under one section. 👍